### DISCUSSION TITLE
The value 'extruder1' is not valid for HEATER

### DISCUSSION THREAD

**Discussion by oentie** [jschuh/klipper-macros#177] [Q&A]
I use prusaslicer for multicolor printing with one extruder 
In the gcode there is M104 T0 S200
                                  M104 T1 S200
this gives me The value 'extruder1' is not valid for HEATER and canceled the print 
Is there something wrong with the M104 macro

**Comment by jschuh** [comment]
You need to ensure that your printer is configured as "Single Extruder Multi Material" if you have only one extruder, so that Prusa Slicer knows not index nonexistent extruders.

![image](https://github.com/jschuh/klipper-macros/assets/13139373/2f73a791-09ca-470d-9003-bc8586d2ac7b)
